web/client/components/geostory/layouts/sections/enhancers/immersiveBackgroundManager.jsx

```diff
@@ -79,6 +86,6 @@ export default compose(
     }),
     backgroundProp,
     withHandlers({
-        updateBackground: ({ sectionId, backgroundId, update = () => { } }) => (path, ...args) => update(`sections[{id: ${sectionId}}].contents[{id: ${backgroundId}}].background.` + path, ...args)
+        updateBackground: ({ sectionId, contentId, update = () => { } }) => (path, ...args) => update(`sections[{"id": "${sectionId}"}].contents[{"id": "${contentId}"}].background.` + path, ...args)
```

Comment:
It seems your changes are correct but I wondering why the updateBackground was there even if it was not used before.
I propose to merge the handler with backgroundProp to create a single function to apply to sections that do not need stream to update background in view.